service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ManageLocalAuthoritiesController.java

```diff
@@ -198,9 +200,18 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         }
 
         if (TRANSFER_COURT == action) {
+            Court oldCourt = caseData.getCourt();
             Court courtTransferred = service.transferCourtWithoutTransferLA(caseData);
             caseDetails.getData().put(PAST_COURT_LIST_KEY, caseData.getPastCourtList());
             caseDetails.getData().put(COURT_KEY, courtTransferred);
+
+            if (!isEmpty(courtTransferred) && RCJ_HIGH_COURT_CODE.equals(courtTransferred.getCode())) {
+                // transferred to the high court -> turn off sendToCtsc
+                caseDetails.getData().put("sendToCtsc", YesNo.NO.getValue());
+            } else if (!isEmpty(oldCourt) && RCJ_HIGH_COURT_CODE.equals(oldCourt.getCode())) {
```

Comment:
what would happen if we got to here and courtTransferred was empty?

Reply:
added extra condition just incase!